src/components/dashboard/Claim.js

```diff
@@ -271,23 +276,20 @@ const Claim = props => {
         </Section.Stack>
         <Section.Stack style={styles.extraInfo}>
           <AnimationsJumpingPeople isCitizen={isCitizen} />
-          {!isCitizen && (
```

Comment:
As I remember one more Claim button with countdown timer should be shown in case the user is not verified/whitelisted. Isn't it?

Reply:
fixed